src/patternfly/components/Nav/nav.scss

```diff
@@ -578,8 +579,6 @@
     --pf-c-nav__link--PaddingRight: var(--pf-c-nav--m-horizontal__link--PaddingRight);
     --pf-c-nav__link--PaddingBottom: var(--pf-c-nav--m-horizontal__link--PaddingBottom);
     --pf-c-nav__link--PaddingLeft: var(--pf-c-nav--m-horizontal__link--PaddingLeft);
-    --pf-c-nav__link--Right: var(--pf-c-nav--m-horizontal__link--Right);
-    --pf-c-nav__link--Left: var(--pf-c-nav--m-horizontal__link--Left);
```

Comment:
Technically this is breaking. While we don't use it, a consumer could in their own overrides/customization. Can you just comment the `--pf-c-nav__link--Right/Left` vars to be removed in a breaking change?